impl/client/src/main/javascript/web/pentaho/visual/views/echarts/Abstract.js

```diff
@@ -35,6 +35,14 @@ define([
     fontSize: 12,
     fontColor: "#333333",
 
+    selectionStates: {
+      NoneSelected: 0,
+      AnySelected: 1,
+      IsSelected: 2
+    },
```

Comment:
Is this enum needed to for sub-classes?

If not, consider declaring this "enum" after `use strict`, above:
```js
var SelectionStates = {
  NoneSelected: 0,
  AnySelected: 1,
  IsSelected: 2
};
```

Otherwise, leave it there, but make it Pascal-case, like is common for "types".

Reply:
Moved it after `use strict`.

Reply:
> Otherwise, leave it there, but make it **Pascal-case**, like is common for "types".

As written above:
```js
var SelectionStates = {
  NoneSelected: 0,
  AnySelected: 1,
  IsSelected: 2
};
```

Reply:
Now you moved to the prototype again.
I was only pointing to preferably using pascal case!